libeos-payg/manager-service.c

```diff
@@ -993,12 +993,19 @@ epg_manager_service_manager_add_code (EpgManagerService     *self,
   const gchar *code_str;
   g_variant_get (parameters, "(&s)", &code_str);
 
+  g_message ("Trying to enter code %s", code_str);
   epg_provider_add_code (self->provider, code_str, &time_added, &local_error);
 
   if (local_error != NULL)
-    g_dbus_method_invocation_return_gerror (invocation, local_error);
+    {
+      g_warning ("Failed to enter code: %s", local_error->message);
```

Comment:
Is this warning-worthy?

```suggestion
      g_warning ("Failed to enter code %s: %s", code_str, local_error->message);
```

(I know it's redundant with the line above.)

Reply:
> Is this warning-worthy?

I think so, as in this is not the expected side of the branch. Another way of looking at is that it is just a warning, not a critical warning or an error. But I'm happy to reduce if you think it doesn't make sense.

I'll squash the suggested change to the original commit and force-push.